lib/roby/plan.rb

```diff
@@ -1609,6 +1609,17 @@ def finalize_event(event, timestamp = nil)
             event.finalized!(timestamp)
         end
 
+        # Copy plan marks from one into the other, additive only i.e. it does not remove
+        # marks from the destination task
+        #
+        # @param from_task The task whose marks will be copyed
+        # @param to_task The task who will be marked just as replaced_task
+        def copy_marks!(from_task:, to_task:)
+            add_permanent_task to_task if permanent_task?(from_task)
+
+            add_mission_task to_task if mission_task? from_task
```

Comment:
Parenthesis. No parenthesis... Choose one. (The convention in this codebase is parenthesis)